2213/1/triangle.rb

```diff
@@ -0,0 +1,36 @@
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+# This method smells of :reek:UtilityFunction
+# This method smells of :reek:FeatureEnvy
+def triangle(side_a, side_b, side_c)
```

Comment:
not sure how this can pass koans if you don't raise triangle_error in this method